README.md

```diff
@@ -150,6 +150,20 @@ been written into the [DAG Service][]'s storage mechanism.
 The input's file paths and directory structure will be preserved in the DAG
 Nodes.
 
+### Importer options
+
+In the second argument of the importer constructor you can specify the following options:
+
+* `chunker` (string, defaults to `"fixed"`): the chunking strategy. Now only supports `"fixed"`
+* `chunkerOptions` (object, optional): the options for the chunker. Defaults to an object with the following properties:
+  * `maxChunkSize` (positive integer, defaults to `262144`): the maximum chunk size for the `fixed` chunker.
+* `strategy` (string, defaults to `"balanced"`): the DAG builder strategy name. Supports:
+  * `flat`: flat list of chunks
+  * `balanced`: builds a balanced tree
+  * `trickle`: builds [a trickle tree](https://github.com/ipfs/specs/pull/57#issuecomment-265205384)
+* `maxChildrenPerNode` (positive integer, defaults to `172`): the maximum children per node for the `balanced` and `trickle` DAG builder strategies
+* `layerRepeat` (positive integer, defaults to 4): (only applicable to the `trickle` DAG builder strategy). The maximum repetition of parent nodes for each layer of the tree.
+
```

Comment:
@dignifiedquire wanna push a commit with jsdoc to this PR so that unixfs-engine gets fresh docs with this refactor? Or perhaps just to a PR to the PR, that would be also good :)